template/.drone.star

```diff
@@ -35,276 +33,236 @@ def testing():
       'arch': 'amd64',
     },
     'steps': [
+      {
+        'name': 'staticcheck',
+        'image': 'golang:1.13',
+        'pull': 'always',
+        'commands': [
+          'go run honnef.co/go/tools/cmd/staticcheck ./...',
+        ],
+        'volumes': [
+          {
+            'name': 'gopath',
+            'path': '/go',
+          },
+        ],
+      },
+      {
+        'name': 'lint',
+        'image': 'golang:1.13',
+        'pull': 'always',
+        'commands': [
+          'go run golang.org/x/lint/golint -set_exit_status ./...',
+        ],
+        'volumes': [
+          {
+            'name': 'gopath',
+            'path': '/go',
+          },
+        ],
+      },
       {
         'name': 'vet',
         'image': 'golang:1.13',
         'pull': 'always',
         'commands': [
-          'go version',
-          'go vet ./...'
+          'go vet ./...',
         ],
         'volumes': [
           {
             'name': 'gopath',
-            'path': '/go'
-          }
-        ]
+            'path': '/go',
+          },
+        ],
       },
       {
         'name': 'test',
         'image': 'golang:1.13',
         'pull': 'always',
         'commands': [
-          'go version',
```

Comment:
I still feel like if we have a major minor golang version that printing out the actual version is potentially useful information.

Honestly maybe even a `go env` might be a good thing to have when building.

Reply:
Still not entirely sure about the factor to be really useful. For the windows builds it would make sense because we can not even see which version gets used as it's not managed by docker images.

Reply:
If the ssh runners have it that's fine by me for a compromise.

Just a heads up I don't know that @bradrydzewski updated the SSH runners for windows to be Go 1.13

Reply:
I have added `go version` and `go env` calls to the linux and windows pipelines, there you go :)